test/middleware/enricher.test.ts

```diff
@@ -0,0 +1,1027 @@
+          },
+          Config: {
+            apiKey: 'dummyApiKey',
+            customMappings: [
+              {
+                from: '0001',
+                to: 'Signup',
+              },
+            ],
+            triggerId: '00XX',
+          },
+          Enabled: true,
+          Transformations: [],
+          // @ts-ignore
+          IsProcessorEnabled: true,
+        },
+        message: {
+          anonymousId: 'ac7722c2-ccb6-4ae2-baf6-1effe861f4cd',
+          channel: 'web',
+          context: {
+            app: {
+              build: '1.0.0',
+              name: 'RudderLabs JavaScript SDK',
+              namespace: 'com.rudderlabs.javascript',
+              version: '1.1.1-rc.2',
+            },
+            library: {
+              name: 'RudderLabs JavaScript SDK',
+              version: '1.1.1-rc.2',
+            },
+            locale: 'en-GB',
+            os: {
+              name: '',
+              version: '',
+            },
+            page: {
+              path: '/tests/html/index4.html',
+              referrer: '',
+              search: '',
+              title: '',
+              url: 'http://localhost/tests/html/index4.html',
+            },
+            screen: {
+              density: 2,
+            },
+            userAgent:
+              'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.163 Safari/537.36',
+          },
+          integrations: {
+            All: true,
+          },
+          messageId: 'fad9b3fb-5778-4db3-9fb6-7168b554191f',
+          originalTimestamp: '2020-04-17T14:42:44.722Z',
+          receivedAt: '2020-04-17T20:12:44.758+05:30',
+          request_ip: '[::1]:53513',
+          sentAt: '2020-04-17T14:42:44.722Z',
+          traits: {
+            age: 23,
+            email: 'testmp@rudderstack.com',
+            firstname: 'Test Kafka',
+          },
+          timestamp: '2020-04-17T20:12:44.758+05:30',
+          type: 'identify',
+          userId: 'user12345',
+        },
+      },
+      {
+        destination: {
+          ID: '1afjtc6chkhdeKsXYrNFOzR5D9v',
+          Name: 'Autopilot',
+          DestinationDefinition: {
+            ID: '1afjX4MlAucK57Q0ctTVlD27Tvo',
+            Name: 'AUTOPILOT',
+            DisplayName: 'Autopilot',
+            Config: {
+              cdkEnabled: true,
+              excludeKeys: [],
+              includeKeys: [],
+            },
+          },
+          Config: {
+            apiKey: 'dummyApiKey',
+            customMappings: [
+              {
+                from: '0001',
+                to: 'Signup',
+              },
+            ],
+            triggerId: '00XX',
+          },
+          Enabled: true,
+          Transformations: [],
+          // @ts-ignore
+          IsProcessorEnabled: true,
+        },
+        message: {
+          anonymousId: 'ac7722c2-ccb6-4ae2-baf6-1effe861f4cd',
+          channel: 'web',
+          context: {
```

Comment:
As we are not asserting other attributes & that's not the scope in unit test here, can keep the context object simple by restricting to only `context.geo` or empty context

Reply:
Makes sense. Incorporated the suggestions